doc/operation-and-maintenance/System-Metrics-Privacy-Policy.md

```diff
@@ -0,0 +1,89 @@
+# Introduction
+MongooseIM system metrics are gathered to analyse the trends and needs of our users, improve MongooseIM, and know where to focus our efforts.
+This section is devoted to explaining on how customise, read, enable and disable collecting of the system metrics.
+
+# Consent
+To ensure the transparency of the functionality being enabled, a log message is generated on every MongooseIM node start (unless metrics service is configured with `report` option).
+The user is being notified that the metrics are gathered and has the right to withdraw the consent at any time without limiting functionalities of the product.
+For more information on how to disable this feature, please see the [Services](Services.md#service_mongoose_system_metrics) section.
+
+# What information is being gathered?
+While developing this feature it is crucial for us to be fully transparent as to what information is being gathered.
+In general, we capture information on how MongooseIM is being used, its version and with what feature set.
+We only report the names of known modules and APIs that are part of opensource and count customisations without disclosing any details.
+User can view all the information that is shared in two different ways. Log file `???LOG FILE NAME FROM JIRA TASK???` contains the most recent report that was sent.
+Moreover, user can configure Tracking ID for their own Google Analytics account and have a view on their MongooseIM status in that dashboard.
+For more information on how to set up the Tracking ID, please see [How to configure additional and private Tracking ID in Google Analytics.](#How-to-configure-additional-and-private-Tracking-ID-in-Google-Analytics?).
```

Comment:
markdown requires the section name to be all small letters, and the question mark is not allowed
```suggestion
For more information on how to set up the Tracking ID, please see [How to configure additional and private Tracking ID in Google Analytics.](#how-to-configure-additional-and-private-tracking-id-in-google-analytics).
```